bin/ubpf-assembler

```diff
@@ -20,7 +20,7 @@ import ubpf.assembler
 def main():
     parser = argparse.ArgumentParser(description=__doc__, formatter_class=argparse.RawDescriptionHelpFormatter)
     parser.add_argument('input', type=argparse.FileType('r'), default='-')
-    parser.add_argument('output', type=argparse.FileType('w'), default='-')
+    parser.add_argument('output', type=argparse.FileType('wb'), default='-')
```

Comment:
This change won't work for e.g., `python3 bin/ubpf-assembler tests/tcp-port-80/tcp-port-80.asm -`.

Reply:
Thanks for pointing it out. I missed that. We need to use `sys.stdout.buffer.write()` to write bytes in Python3. https://docs.python.org/3.7/library/sys.html#sys.stdout